docs/DeveloperGuide.md

```diff
@@ -882,41 +903,42 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | first-time user                            | access the available commands  | use the app efficiently                |
 | `* *`    | user                                     | save and load my Module data   | keep track of my Modular progress                                            |
 
-*{More to be added}*
 
 ### Use cases
 
 (For all use cases below, the **System** is the `GradPad` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: UC01 - Delete a Module from `Completed Modules`**
+**Use case: UC01 - Delete a module from `Completed Modules`**
 
 **MSS**
 
 1.  User requests to delete a specific Module in the `Completed Modules`
 2.  GradPad deletes the module
-3.  GradPad displays the deleted module onto the `Command Line Display`
+3.  GradPad displays the deleted module onto the `Result Display`
 
     Use case ends.
 
 **Extensions**
 
 * 1a. The module does not exist in `Completed Modules`.
 
-  Use case ends.
+    * 1a1. GradPad shows an error message.
+    
+        Use case ends.
 
 * 2a. The given module code is invalid.
 
     * 2a1. GradPad shows an error message.
 
       Use case resumes at step 1.
       
-**Use case : UC02 - Add a Module into `Completed Modules`**
+**Use case : UC02 - Add a module into `Completed Modules`**
 
 **MSS**
 
 1. User requests to add a module into the `Completed Modules`
 2. GradPad adds the module into `Completed Modules`
-3. GradPad displays the module added onto the `Command Line Display`
+3. GradPad displays the module added onto the `Result Display`
 
     Use case ends.
```

Comment:
I think we can add an extension when users try to add a module that doesn't exist in NUS